postgres-express-setup-tutorial.md

```diff
@@ -91,56 +105,66 @@ In order to list all your databases, you can type `\list`. Your new database sho
 
-Before you can implement the database usage in your Node.js application, you have to install sequelize and pg (postgres client for Node.js) on the command line by using npm:
+For this application, we'll use {{% a_blank "Sequelize" "https://github.com/sequelize/sequelize.git" %}}, as it supports multiple dialects, one of which is PostgreSQL. Sequelize provides a comfortable API to work with PostgreSQL databases from setup to execution, but there are [many ORMs to choose](https://en.wikipedia.org/wiki/List_of_object-relational_mapping_software) from if you want to expand your toolbelt.
+
+Before you can implement database usage in your Node.js application, install sequelize and pg (postgres client for Node.js) on the command line  using npm:
 
 {{< highlight javascript >}}
 npm install pg sequelize --save
 {{< /highlight >}}
 
-The following use case implements a database for a Twitter clone application where you have the two models Author and Tweet. Therefore, you usually have a folder in your Node.js application called *src/models/*. Its content are files for each model in your database (e.g. *src/models/author.js* and *src/models/tweet.js*). Moreover, there is one file (e.g. *src/models/index.js*) which combines all models and exports all the necessary information to the Express server.
+[placeholder:UML related header?]
+
+The following case implements a database for a Twitter clone application with two models: User and Message. Usually, there is a folder in your Node.js application called *src/models/* that contains files for each model in your database (e.g. *src/models/user.js* and *src/models/message.js*). There is also a file (e.g. *src/models/index.js*) that combines all models and exports all the necessary information to the Express server.
 
-Let's start with the two models in the *src/models/[modelname].js* files. The Author model could look like the following:
+Before diving into the code for your application, it's always a good idea to map the relationships between objects and how to handle the data that must pass between them.  A UML diagram (short for Unifified Modeling Language) is a straightforward way to express relationships between entities in a way that can be referenced quickly as you type them out. This is useful for the person laying the groundwork for an application as well as anyone who wants to add to it. For our Twitter clone application, the UML diagram would appear as such:
+
+[placeholder: UML image]
```

Comment:
You can leave it like this, I will add later the diagram here. You can find the diagram in your inbox though.